src/Microsoft.NetCore.Analyzers/Core/ImmutableCollections/DoNotCallToImmutableCollectionOnAnImmutableCollectionValue.Fixer.cs

```diff
@@ -19,10 +26,70 @@ public sealed override FixAllProvider GetFixAllProvider()
             return WellKnownFixAllProviders.BatchFixer;
         }
 
-        public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
+        private static readonly ImmutableArray<string> ToImmutableMethodNames = new[]
```

Comment:
Consider share this list between analyzer and codefix, so we don't have to keep them in sync.